gumby/scenario.py

```diff
@@ -209,21 +233,21 @@ def _parse_peerspec(self, peerspec):
 
         return yes_peers, no_peers
 
-    def _parse_for_this_peer(self):
+    def _parse_for_this_peer(self, peerspec):
         raise NotImplementedError('override this method please')
 
-    def _preprocess_line(self, line):
+    def _expand_line(self, line):
         # Look for $VARIABLES to replace with config options from the env.
         for substitution in self._re_substitution.findall(line):
             if substitution[1:] in environ:
                 line = line.replace(substitution, environ[substitution[1:]])
 
         return line
 
-class ScenarioRunner(ScenarioParser):
 
+class ScenarioRunner(ScenarioParser):
     """
-    Reads, parses and schedules events from scenario file.
+    Reads, parses and schedules events from scenario files.
```

Comment:
Well you can #include them, so even though they form one whole, they come from multiple file.